package.json

```diff
@@ -20,55 +20,52 @@
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:cucumber": "cucumber-js",
-    "start:pm2": "pm2 --name Backend start pnpm -- start:prod",
+    "start:pm2": "pm2 --name Backend start dist/main.js",
     "stop:pm2": "pm2 delete 0"
```

Comment:
I think it should be name of the process (Backend) 
example `pm2 delete Backend`